client/lobby/RandomMapTab.cpp

```diff
@@ -145,39 +137,29 @@ RandomMapTab::RandomMapTab()
 		updateMapInfoByHost();
 	});
 
-	// Show random maps btn
-	showRandMaps = std::make_shared<CButton>(Point(54, 535), "RANSHOW", CGI->generaltexth->zelp[252]);
+	buttonShowRandomMaps = std::make_shared<CButton>(Point(54, 535), "RANSHOW", CGI->generaltexth->zelp[252]);
 
-	// Initialize map info object
 	updateMapInfoByHost();
 }
 
 void RandomMapTab::showAll(SDL_Surface * to)
 {
 	CIntObject::showAll(to);
-
 	// Headline
 	printAtMiddleLoc(CGI->generaltexth->allTexts[738], 222, 36, FONT_BIG, Colors::YELLOW, to);
```

Comment:
All this should be replaced with CLabel

Reply:
I wonder if we can do something about ```IFont::renderTextLeft``` usage too, though it used to render on top of ```SDL_Surface```.